stoservice/src/main/java/com/tutrit/stoservice/StoserviceApplication.java

```diff
@@ -16,6 +17,7 @@ public class StoserviceApplication {
     public static void main(String[] args) {
         SpringApplication.run(StoserviceApplication.class, args);
         ApplicationContextLoader.run();
+        if (args.length > 0) get(ArgsSetup.class).readingFlags(args);
```

Comment:
The "if" condition must be moved inside the "readingFlags()" method.
You will get an NullPointerException if the args is null

Reply:
The main method does not create null, it creates an expression (with an empty argument)
new String[0].
foreach will take an empty value without any problem.
```
String[] str = new String[0];
        System.out.println(str.length);
        for (String a:str) {
            System.out.println(a);
        }
```
This construct does not create exceptions